setup/flex.rst

```diff
@@ -33,25 +33,30 @@ name. However, if the application has Symfony Flex installed, that command ends
 up installing and enabling the SwiftmailerBundle, which is the best way to
 integrate Swiftmailer, the official mailer for Symfony applications.
 
-When Symfony Flex is installed in the application and you execute ``composer require``,
-the application makes a request to Symfony Flex servers before trying to install
-the package with Composer:
+When Symfony Flex is installed in the application and you execute ``composer
+require``, the application makes a request to Symfony Flex servers before
+trying to install the package with Composer:
 
 * If there's no information about that package, Flex server returns nothing and
   the package installation follows the usual procedure based on Composer;
+
 * If there's special information about that package, Flex returns it in a file
-  called "recipe" and the application uses it to decide which package to install
-  and which automated tasks to run after the installation.
+  called "recipe" and the application uses it to decide which package to
+  install and which automated tasks to run after the installation.
 
 In the above example, Symfony Flex asks about the ``mailer`` package and the
-Symfony Flex server detects that ``mailer`` is in fact an alias for SwiftmailerBundle
-and returns the "recipe" for it.
+Symfony Flex server detects that ``mailer`` is in fact an alias for
+SwiftmailerBundle and returns the "recipe" for it.
+
+Flex keeps tracks of which recipes it installed in a ``symfony.lock`` file that
+must be committed in your code repository.
```

Comment:
I'd rewrite to:
`Flex keeps tracks of installed recipes in a ``symfony.lock`` file which must be committed in your code repository.`
Just for better concordance